project/style.css

```diff
@@ -0,0 +1,227 @@
+.login-section{
+  margin-left: 15%;
+  margin-right:15%;
+  margin-top: 35%;
+}
+
+.section-image1{
+  width:30%;
+  float:left;
+}
+
+.section-image2{
+  height:25%;
+  width:25%;
+  float:right;
+  margin-left: 52%;
+  position:absolute;
+}
+
+.right-paragraph {
+  margin-left: 25%;
+  float: right;
+  position:absolute;
+  width: 40%;
+}
+
+.left-paragraph {
+  float: left;
+  position: absolute;
+  margin-right: 25%;
+  width: 40%;
+}
+
+.header-form{
+  background-color: rgb(51, 153, 255);
+  color: rgb(255, 255, 255);
+  box-shadow: 0px 10px 30px rgb(166, 166, 166);
+  height: 20%;
+  margin-top: 0%;
+  text-align: center;
+}
+
+br { font-size: 200%; }
+
+.content-form{
+  width:30%;
+  margin-top: -30%;
+  margin-left: 20%;
+  border: 2px solid black;
+}
+
+.login-form{
+  position: absolute;
+  background-color:
+  rgb(255,255,255);
+  width: 20%;
+  height: 40%;
+}
+
+input[type=text], [type=date] {
+    width: 30%;
+    display: inline-block;
+    border-width:0px;
+    outline: 0;
+    border-width: 0 0 2px 0;
+    border-color: rgb(166, 166, 166);
+}
+
+input.login-input{
+  width: 90%;
+  border-width:0px;
+  border-width: 0 0 2px 0;
+  font-size: 135%;
+  font-family: roboto;
+  margin: auto;
+  display: flex;
+  border-color: rgb(230, 230, 230);
+}
+
+input[type=radio]{
+  border-color: rgb(166, 166, 166);
+}
+
+input[type=submit]{
+  margin-top: 5%;
+  width: 100px;
+  height: 60px;
+  background-color: rgb(51, 153, 255);
+  box-shadow: 0px 10px 30px rgb(166, 166, 166);
+  float: right;
+  color: white;
+  font-size: large;
+}
+
+.login-button{
+  margin:auto;
+  margin-bottom: 2%;
+  width: 35%;
+  height: 11%;
+  background-color: rgb(51, 153, 255);
+  box-shadow: 0px 10px 30px rgb(166, 166, 166);
+  color: white;
+  font-size: large;
+  display: block;
+  margin-top: 10%;
+}
+
+.content2{
+  margin-top: 10%;
+}
+
+.sv-logo{
+  display: inline-block;
+  width: 12%;
+  height: 10%;
+  margin-left: 5%;
+}
+
+hr {
+  border:none;
+  height: 20px;
+ 	width: 100%;
+	height: 35px;
+	margin-top: 0;
+	box-shadow: 0px 10px 35px -9px rgba(204,194,204,1);
```

Comment:
I'd argue that `0vmin` would be much better in this case. Am I right? Am I just messing around?